app/controllers/admin/projects_controller.rb

```diff
@@ -17,6 +17,7 @@ def destroy
     if Rails.application.config.samson.project_deleted_email
       ProjectMailer.deleted_email(@current_user, project).deliver_later
     end
+    Samson::Hooks.fire(:audit_action, current_user, 'deleted project', project)
```

Comment:
Had to make it slightly more complex to handle the soft_deletion, but I like it better this way.  Definitely the way to go with  the other app.  

Still need to keep the hooks for certain actions that won't be captured cleanly by the after_commit concern.  IE auth, merge user, deploy start/end, unauthorized actions.